src/libexec/zmdiaglog

```diff
@@ -601,12 +601,22 @@ sub run() {
     );
     cp $_, "$destination/confs" for @conf_files;
 
+    mkdir("$destination/confs/network");
+    my @network_conf_files = (
+        "/opt/zimbra/conf/zextras/config",
```

Comment:
I assume this is a directory.  Does it make sense to include that in the list of things to copy?
Presumably, if none of these files exist, that doesn't pose a problem.  The way I've currently installed a FOSS install, I don't have a `/opt/zimbra/conf/zextras` directory present.  Have you tested what happens under those circumstances?

Reply:
"/opt/zimbra/conf/zextras/config" is a file. Simply it's ignored in a FOSS environment, anyway i've added a check to test if Network-NG is preset.